### ISSUE TITLE
`maputnik.github.io` should use `maplibre.org/maputnik` editor

### ISSUE THREAD

**Issue by nyurik** [maplibre/maputnik#875]
The main [Maputnik site](https://maputnik.github.io/) uses an outdated version of Maputnik - it should redirect to the up-to-date one instead.

**Comment by HarelM** [comment]
I can't find the issue that talks about how to use an older version of Maputnik, but right now, this allows testing of some functionality and see how they behaved in an older version very easily.
While this might be temporary, I still feel the need to keep the old site using the old version to allow easier comparison.
I personally still don't feel very comfortable with latest version, but this might just be me... :-)

**Comment by nyurik** [comment]
I'm ok to host multiple versions, selectable, at one location, but I think it will confuse everyone if there are two different versions in two unconnected places

**Comment by HarelM** [comment]
I think hosting multiple versions is an overhead no one will support (judging from the current maintenance support I see).
I'm OK with leaving it for a few month as is until we (I?) feel that the current version is at least as good as the previous one.
The reason for this is that when I took over the project there were changes that weren't released on one hand and broke stuff on the other hand, so the released version is still a bit shaky from my point of view.
Having said that, I don't have hard feelings either way.

**Comment by HarelM** [comment]
Found the issue: https://github.com/maplibre/maputnik/issues/862

**Comment by vknoppkewetzel** [comment]
Hello! I just commented this on a closed ticket from this week that I found after google (wasn't aware Maputnik had transitioned) directed me towards that particular ticket in my search

It may be useful to have a message pop up for Maputnik users in the old editor site. I wasn't aware there was a transition (yay!) and when trying to dive back into stylework in Maptunik I was confused about the editor not working and assumed there was work being done and kept circling back later (clearly there was!)

Maybe on https://maputnik.github.io/editor/#0.8/0/0 page just a simple "Maputnik has moved to MapLibre! head over to this URL instead: https://maplibre.org/maputnik"
